pkg/cmd/pulumi/package_gen_sdk.go

```diff
@@ -63,6 +91,8 @@ func newGenSdkCommand() *cobra.Command {
 		"The SDK language to generate: [nodejs|python|go|dotnet|java|all]")
 	cmd.Flags().StringVarP(&out, "out", "o", "./sdk",
 		"The directory to write the SDK to")
+	cmd.Flags().StringVarP(&javaBuildTool, "java-build-tool", "", "gradle",
```

Comment:
Assume in the future pu/pu will not be allowed to know anything about java specifically, we'll need to have a different way of setting and passing options like this.